### DISCUSSION TITLE
Gluetun taking down containers

### DISCUSSION THREAD

**Discussion by theliamllyd** [qdm12/gluetun#1716] [Q&A]
I am currently running Gluetun through Docker-Compose, along with the stack I have transmission and prowlarr (for scientific reasons). It appears whenever my public-ip changes through Gluetun it renders the other containers unaccessible. Has anybody else experience or know of a fix for this? Only fix I have found for this is to re-deploy the containers and re-run the compose file. I'm running docker through Docker Desktop for Mac. Using the settings for OpenVPN via PureVPN. I am connected via the environment variable '-SERVER_COUNTRIES=GERMANY'.

**Comment by qdm12** [comment]
This is a well known issue, restarting the gluetun container will leave other containers disconnected. Subscribe to #641

If your case happens without a container restart, and with a vpn internal restart, then please an issue instead of discussion, but I don't believe this is the case here.
